packages/oui-dropdown/README.md

```diff
@@ -0,0 +1,184 @@
+    <oui-dropdown-content>
+      <oui-action-menu>
+        <oui-action-menu-item
+          data-text="Action 1"
+          data-on-click="$ctrl.showCurrentValueInPopup('Action 1')"></oui-action-menu-item>
+        <oui-action-menu-item
+          data-text="Action 2"
+          data-on-click="$ctrl.showCurrentValueInPopup('Action 2')"></oui-action-menu-item>
+        <oui-action-menu-item divider
+          data-text="Lien externe"
+          data-type="external"
+          data-href="#"></oui-action-menu-item>
+      </oui-action-menu>
+    </oui-dropdown-content>
+  </oui-dropdown>
+</div>
+<div style="text-align: center">
+  <oui-dropdown arrow>
+    <button type="button" class="oui-button oui-button_secondary oui-button_icon-only oui-button_small-width" oui-dropdown-trigger>
+      <i class="oui-icon oui-icon-gear_line" aria-hidden="true"></i>
+    </button>
+    <oui-dropdown-content>
+      <oui-action-menu>
+        <oui-action-menu-item
+          data-text="Action 1"
+          data-on-click="$ctrl.showCurrentValueInPopup('Action 1')"></oui-action-menu-item>
+        <oui-action-menu-item
+          data-text="Action 2"
+          data-on-click="$ctrl.showCurrentValueInPopup('Action 2')"></oui-action-menu-item>
+        <oui-action-menu-item divider
+          data-text="Lien externe"
+          data-type="external"
+          data-href="#"></oui-action-menu-item>
+      </oui-action-menu>
+    </oui-dropdown-content>
+  </oui-dropdown>
+</div>
+<div style="text-align: right">
+  <oui-dropdown end arrow>
+    <button type="button" class="oui-button oui-button_secondary oui-button_icon-only oui-button_small-width" oui-dropdown-trigger>
+      <i class="oui-icon oui-icon-gear_line" aria-hidden="true"></i>
+    </button>
+    <oui-dropdown-content>
+      <oui-action-menu>
+        <oui-action-menu-item
+          data-text="Action 1"
+          data-on-click="$ctrl.showCurrentValueInPopup('Action 1')"></oui-action-menu-item>
+        <oui-action-menu-item
+          data-text="Action 2"
+          data-on-click="$ctrl.showCurrentValueInPopup('Action 2')"></oui-action-menu-item>
+        <oui-action-menu-item divider
+          data-text="Lien externe"
+          data-type="external"
+          data-href="#"></oui-action-menu-item>
+      </oui-action-menu>
+    </oui-dropdown-content>
+  </oui-dropdown>
+</div>
+```
+
+## API
+
+### oui-dropdown
+
+The component for a dropdown.
+
+Availability:
+
+ - Element
+
+| Attribute         | Type            | Binding | Values              | Default             | Description                        |
+| ----              | ----            | ----    | ----                | ----                | ----                               |
+| arrow             |                 |         |                     |                     | display the dropdown arrow         |
+| start             |                 |         |                     |                     | modifier for left alignment        |
```

Comment:
Because it has been decided to call it start. (Popper.js uses this notation and it's not just a matter of simple alignment, but border to border alignment...)
It could be problematic here to call it left/right if one day we decide to create the dropdown that pops to  the left/right. (See Popper.js doc)

Reply:
Updated with `align="start|end"`